### ISSUE TITLE
Can't differentiate between floating terminal and editor

### ISSUE THREAD

**Issue by mrprofessor** [voldikss/vim-floaterm#16]
<img width="738" alt="image" src="https://user-images.githubusercontent.com/11652364/67664910-9612d680-f98e-11e9-8be9-1168cd54e2b6.png">

My setup
``` vim
" Floating stuff
noremap  <leader>t  :FloatermToggle<CR>i
noremap! <leader>t  <Esc>:FloatermToggle<CR>i
tnoremap <leader>t  <C-\><C-n>:FloatermToggle<CR>
let g:floaterm_width = 100
let g:floaterm_winblend = 30
let g:floaterm_background = '#000000'
```

The background doesn't seem to work.

**Comment by voldikss** [comment]
What's your color scheme?

**Comment by mrprofessor** [comment]
> What's your color scheme?

gruvbox

**Comment by voldikss** [comment]
Cannot reproduce that issue. Type `:hi Normal` and `:hi NormalFloat` and
see what is the output.

Rudra Narayan <notifications@github.com> 于 2019年10月29日周二 下午2:21写道：

**Comment by mrprofessor** [comment]
```
:hi Normal
Normal         xxx ctermfg=223 ctermbg=235 guifg=#ebdbb2 guibg=#282828
```

and

```
:hi NormalFloat
NormalFloat    xxx links to Pmenu
```

**Comment by voldikss** [comment]
Then `:hi Pmenu`?

**Comment by voldikss** [comment]
I've made a

**Comment by mrprofessor** [comment]
Removing winblend seems to fix the issue.

<img width="1425" alt="image" src="https://user-images.githubusercontent.com/11652364/67830256-f6755580-faff-11e9-897c-4257fbe564da.png">


I think it should be another issue but everytime I open the terminal, It is appending an `i` char

**Comment by voldikss** [comment]
> Removing winblend seems to fix the issue.
> 
> <img alt="image" width="1425" src="https://user-images.githubusercontent.com/11652364/67830256-f6755580-faff-11e9-897c-4257fbe564da.png">

It's not fixed because the background color of the floating window was not `#000000`

Sorry I still can not figure out what causes this issue, but here are the codes which deal with the background color of the floating window. You may add some codes such as `echo` to debug it if you want.

https://github.com/voldikss/vim-floaterm/blob/6ebb2b04f14f85c875a4af4f9e62da5ed6ba8fe4/autoload/floaterm.vim#L206-L208

**Comment by voldikss** [comment]
@mrprofessor Excuse me, may I ask if this issue still exists?

**Comment by mrprofessor** [comment]
> @mrprofessor Excuse me, may I ask if this issue still exists?

I haven't updated for a long time. 


I just checked out to this commit.
https://github.com/voldikss/vim-floaterm/commit/6ffb68a79031a1a797f63e403b8deb36c597b17c
 
It fixes both #17 and #16 . Will definitely help debug when I get any breathing space from my work. Thanks for creating the plugin though. @voldikss

_Originally posted by @mrprofessor in https://github.com/voldikss/vim-floaterm/issues/17#issuecomment-549060731_

Will check and update here.@voldikss

**Comment by mrprofessor** [comment]
![image](https://user-images.githubusercontent.com/11652364/71999199-fcf21d00-3266-11ea-9e27-f669f100d7b8.png)
Works just fine with the borders.
